README.md

```diff
@@ -1,142 +1,45 @@
-cd /opt/gfxinstall
-./sgx-install.sh
-sed -i 's:/sys/devices/ocp*/56000000.sgx:/sys/devices/platform/ocp*/56000000.sgx:' /etc/init.d/sgx-startup.sh
-```
-pvrsrvkm should now be shown in lsmod
-
-
-**cogl 1.22.0:**
-```
-cd /usr/src
-wget http://ftp.gnome.org/pub/GNOME/sources/cogl/1.22/cogl-1.22.2.tar.xz
-tar xf cogl-1.22.2.tar.xz
-cd cogl-1.22.2/
-./configure --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/ --enable-introspection --disable-gles1 --disable-cairo --disable-gl --enable-gles2 --enable-null-egl-platform --enable-cogl-pango
-sed -i 's/#if COGL_HAS_WAYLAND_EGL_SERVER_SUPPORT/#ifdef COGL_HAS_WAYLAND_EGL_SERVER_SUPPORT/' cogl/winsys/cogl-winsys-egl.c 
-make
-make install 
-```
+- Toggle is Python3 compatible.
+- Code is formatted using YAPF, check the setup.cfg for style.
+- Travis is set up to check formatting only for now. Use yapf3.
 
-**libinput-1.0.0**
-```
-cd /usr/src
-wget http://www.freedesktop.org/software/libinput/libinput-1.0.0.tar.xz
-./configure --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/
-make
-make install
-```
-
-**glib-2.48.2**
-Glib from apt gets installed in /lib, so library path must be set from here on. 
-```
-cd /usr/src
-wget http://ftp.gnome.org/pub/gnome/sources/glib/2.48/glib-2.48.2.tar.xz
-tar xf glib-2.48.2.tar.xz
-cd glib-2.48.2/
-./configure --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/
-make
-make install
-export LD_LIBRARY_PATH=/usr/lib/arm-linux-gnueabihf/
-```
-
-**Clutter 1.26**
-Note: Clutter 1.26 requires Glib >=2.44 which is not installed on Debian Jessie. 
-```
-cd /usr/src
-wget http://ftp.acc.umu.se/pub/GNOME/sources/clutter/1.26/clutter-1.26.0.tar.xz
-tar xf clutter-1.26.0.tar.xz
-cd clutter-1.26.0
-LD_LIBRARY_PATH=/usr/lib/arm-linux-gnueabihf/ ./configure --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/ --disable-x11-backend  --enable-egl-backend --enable-evdev-input --disable-gdk-backend
-make
-make install
-```
-
-**Mx**
-Note: the pkg-config files were installed in the standard 
-```
-cd /usr/src
-git clone https://github.com/clutter-project/mx.git
- ./autogen.sh --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/ --with-winsys=none --disable-gtk-doc --enable-introspection 
-make
-make install
-```
+**Install Toggle**
 
-**Mash (with STL-import)**
-Note: Disable compiling the examples and 
-use the right --library format for the g-ir-scanner
 ```
-cd /usr/src/
-git clone https://github.com/eliasbakken/mash.git
-cd /usr/src/mash
-./autogen.sh --prefix=/usr --libdir=/usr/lib/arm-linux-gnueabihf/ --enable-introspection
-sed -i 's/--library=mash-@MASH_API_VERSION@/--library=mash-@MASH_API_VERSION@/ --library-path=/usr/src/mash/mash/.libs/' mash/Makefile.am
-make
-make install
+apt install python-setuptools python-gi
 ```
 
-**Toggle**
 ```
-cd /usr/src
-git clone https://intelligentagent@bitbucket.org/intelligentagent/toggle.git
+git clone https://github.com/intelligent-agent/toggle
 cd toggle
-make install
+pip3 install -e . -r requirements

```

Comment:
We may want to add a caveat somewhere that explains this is all based on debian-style distros. If you're using arch or some other distro base `pip3` is actually `pip` and the `apt install` will be something else also.

Reply:
pip3 should point to the py3 version of pip in both Ubuntu and Debian. I think if anyone wants to use Arch, they are knowledgeable enough to know what to do hopefully : )